### ISSUE TITLE
ReferenceAssembliesFromCode doesn't respect "//css_ignore_namespace" directive

### ISSUE THREAD

**Issue by oleg-shilo** [oleg-shilo/cs-script#4]
All evaluators don't process "//css_ignore_namespace" when they invok ReferenceAssembliesFromCode. Though only with Mono compiler it leads to the script compile failure.

**Comment by oleg-shilo** [comment]
Fixed with Commit:39da859
Namespaces can now be ignored either from code or via evaluator settings:
```C#
dynamic script = CSScript.MonoEvaluator
                         .With(eval=>eval.DisableReferencingFromCode = true)
                         .LoadCode("...
```

**Comment by oleg-shilo** [comment]
Closed as Fixed
